### PR TITLE
i105 export django model data

### LINKED ISSUE #105: Export django model data
## Basic Information
From the admin dashboard, they should be able to export data to a spreadsheet for each endpoint
- We shouldnt include passwords in the exported data
- We shouldnt include any user data that doesnt have consent! This should also apply for foreign key relations as well for that user

### PR DESCRIPTION
**Pull request by makon1234**
## Change Summary
Can now export user data from django endpoints. Password not included and users without consent are not exported. 

### Change Form
na

- [x] The pull request title has an issue number
- [x] The change works by "Smoke testing" or quick testing
- [ ] The change has tests
- [ ] The change has documentation

## Other Information
no

# Related Issue

- Resolve #105

### CODE REVIEW

File: server/api/actions.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+def export2csv(modeladmin: ModelAdmin, request: HttpRequest, queryset: QuerySet):
+    import csv
+
+    response = HttpResponse(
+        content_type="text/csv",
+        headers={"Content-Disposition": f'attachment; filename="{modeladmin.model._meta.model_name}s.csv"'},
+    )
+
+    writer = csv.writer(response)
+    replacements = {'team_id': 'team_id__name', 'user': 'user__frist_name'}
```
Comment (ChantelleKerr) [inline server/api/actions.py:18]: it should be "user__first_name"

**Review by ChantelleKerr — CHANGES_REQUESTED** [review]
I gget a few errors

Exporting mileage data:
Screen Shot 2023-07-16 at 9 46 20 am

Exporting Team data:
Screen Shot 2023-07-16 at 9 46 05 am

File: server/api/actions.py (hunk 1 above)
Comment (makon1234) [reply server/api/actions.py]: my bad, its been fixed

**Review by ChantelleKerr — APPROVED** [review]
Great job! Approved

**Comment by makon1234** [comment]
Hi, I don't think I have fixed all the errors yet, I still need to do exporting team data, that one is easy but in the Users.csv, you want team name instead of team_id right, right now it changes all the team_id so.... we still need work on that

**Comment by ChantelleKerr** [comment]
> Hi, I don't think I have fixed all the errors yet, I still need to do exporting team data, that one is easy but in the Users.csv, you want team name instead of team_id right, right now it changes all the team_id so.... we still need work on that

oops, my bad. Let me know when its all good

**Comment by makon1234** [comment]
Hi, it should be all good now. Could you please confirm?

**Review by ChantelleKerr — APPROVED** [review]
Works well! Good job :)

Approved.

**Pull request merged**